### PR TITLE
feat: new viz service

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

- add a skeleton next JS (app router) app for the new `viz` service
- add the infra files, including a managed certificate and ingress rules for this new service
- add a Dockerfile and a deployment GH action for the new service

## Risk

N/A (doesn't affect any prod system)

## Deploy Plan

- apply infra (new deployment will be stuck in PullImageErr since the image doesn't exist yet)
- run the new github action to deploy
- monitor the new managed cert until it gets provisioned

### CODE REVIEW

File: viz/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import type { Metadata } from "next";
+import { Inter } from "next/font/google";
+import "./globals.css";
+
+const inter = Inter({ subsets: ["latin"] });
+
+export const metadata: Metadata = {
+  title: "Create Next App",
```
Comment (flvndvd) [inline viz/app/layout.tsx:8]: ✂️ ?

File: viz/tailwind.config.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import type { Config } from "tailwindcss";
+
+const config: Config = {
+  content: [
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
```
Comment (flvndvd) [inline viz/tailwind.config.ts:5]: This does not seem to be targeting the right path for the app router hierarchy.

File: k8s/services/viz-service.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    cloud.google.com/backend-config: '{"default": "viz-backendconfig"}'
+spec:
+  selector:
+    app: viz
+    name: viz-pod
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 3007
+  type: ClusterIP
```
Comment (flvndvd) [inline k8s/services/viz-service.yaml:15]: Just to make sure, this limits the accessibility of this service to the cluster, right?

**Review by flvndvd — APPROVED** [review]
LGTM

File: viz/tailwind.config.ts (hunk 2 above)
Comment (fontanierh) [reply viz/tailwind.config.ts:5]: indeed

File: k8s/services/viz-service.yaml (hunk 3 above)
Comment (fontanierh) [reply k8s/services/viz-service.yaml:15]: This is just the type of resource we create for our services. It gives the service an IP accessible within the cluster (which is then targetted by our ingress / external load balancer)

**Pull request merged**